apps/src/blockly/addons/cdoXml.js

```diff
@@ -273,6 +273,34 @@ export function addNameToBlockFunctionCallBlock(blockElement) {
   }
 }
 
+/**
+ * If a field should was serialized before we had behavior ids, manually add
+ * one based on the behavior name found in the field.
+ *
+ * @param {Element} blockElement - The XML element for a single block.
+ */
+function addMissingBehaviorId(blockElement) {
+  const blockType = blockElement.getAttribute('type');
+  if (blockType === BLOCK_TYPES.behaviorGet) {
+    setIdFromTextContent(getFieldOrTitle(blockElement, 'VAR'));
+  } else if (blockType === BLOCK_TYPES.behaviorDefinition) {
+    setIdFromTextContent(getFieldOrTitle(blockElement, 'NAME'));
+  } else {
```

Comment:
nit: do we need this `else` since we will just exit out of the function now anyway?

Reply:
Nope, we don't! Will remove.